source/includes/cli-guide/_cli-overview.html.md

```diff
@@ -1,3 +1,21 @@
 # CLI Overview
 
-The Bonsai Command Line Interface (CLI) is a text-based tool that enables you to configure and control the Bonsai Artificial Intelligence Engine. The CLI is especially useful for automation and connection to other tools. Currently, there are some actions that can only be performed using the CLI, such as loading your Inkling file and connecting your simulator. 
\ No newline at end of file
+```powershell
```

Comment:
The two tabs capture Python 2 and Python 3 instructions in both Windows and macOS which all have subtle differences except that Windows is the same for both Python 2 and 3, but there doesn't seem to be a good way to capture 3 cases, just 2 or 4.